lms/envs/common.py

```diff
@@ -61,6 +61,7 @@
 }
 
 LMS_ROOT_URL = "http://localhost:8000"
+ENTERPRISE_API_URL = 'http://localhost:8000/enterprise/api/v1/'
```

Comment:
@bdero nit: LMS_ROOT_URL + "/enterprise/api/v1/" to indicate that it is actually running in the LMS? Matter of taste though.